.claude/database-ops/SCHEMA.md

```
@@ -0,0 +1,206 @@
+Engine: `ReplacingMergeTree(version)` | Partition: `toYYYYMMDD(timestamp)`
+
+| Column | Type | Notes |
+|---|---|---|
+| id, tenant_id, environment_id, external_customer_id, event_name | String | |
+| customer_id, subscription_id, sub_line_item_id, price_id, feature_id | String | |
+| meter_id | Nullable(String) | |
+| period_id | UInt64 | billing period start (epoch-ms) |
+| timestamp, ingested_at, processed_at | DateTime64(3) | |
+| qty_total | Decimal(25,15) | |
+| unique_hash | Nullable(String) | |
+| version | UInt64 | sign | Int8 | |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)